CHANGELOG

```diff
@@ -19,11 +22,6 @@ v3.10.12 (XXXX-XX-XX)
   connection statistics were built up in memory, but were never released because
   the statistics background thread was not running.
 
-* Avoid memory leak in case an arangod instance is started with the option 
```

Comment:
This does not look right to remove an older CHANGELOG entry. I think this is a merge mistake.

Reply:
TBH, I don't know what's going on here. 
In the local version of the branch this CHANGELOG entry is not removed, and my local version is up-to-date. There is nothing left to push.
Maybe this is just a Github UI issue. :shrug:

Reply:
OK, let's ignore it.